package.json

```diff
@@ -34,7 +34,7 @@
     "koa-route": "^3.2.0",
     "mermaid": "^8.13.4",
     "pako": "^2.0.4",
-    "puppeteer": "^11.0.0"
+    "puppeteer": "^13.1.1"
```

Comment:
I just upgraded it to `^13` in other commit, could you please resolve the merge conflict?